### DISCUSSION TITLE
Always has an unread message

### DISCUSSION THREAD

**Discussion by zgcbj** [community/community#46577] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/1614439/217131262-11021377-87ed-45b0-987b-4d6d70f8709f.png)

**Comment by radar-blip** [comment, marked as answer]
The answer in this thread should help: https://github.com/orgs/community/discussions/6874#discussioncomment-3593547
